packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+import Cocktail from 'cocktail';
+import FormView from '../form';
+import SettingsPanelMixin from '../mixins/settings-panel-mixin';
+import Template from 'templates/settings/subscription.mustache';
+
+const View = FormView.extend({
+  template: Template,
+  className: 'subscription',
+  viewName: 'settings.subscription',
+
+  events: {
+    'click button': 'submit',
+  },
+
+  initialize (options = {}) {
+    this._config = Object.assign({}, {
+      managementClientId: '98e6508e88680e1a',
```

Comment:
This is the content server's client_id, let's use a value of `59cceb6f8c32317c` for the subscriptions page.